spec/poros/food_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+describe Food do
+  it "exists and has attributes" do
+    attributes = {
+      description: "sweet potatoes",
+      gtinUpc: "492111402857",
+      brandOwner: "ARCHER FARMS",
+      ingredients: "SWEET POTATOES."
+    }
+
+    food_info = Food.new(attributes)
+
+    expect(food_info).to be_an_instance_of(Food)
+    expect(food_info.upc).to eq("492111402857")
+    expect(food_info.description).to eq("sweet potatoes")
+    expect(food_info.brand_owner).to eq("ARCHER FARMS")
+    expect(food_info.ingredients).to eq("SWEET POTATOES.")
+  end
+end
```

Comment:
I like that you remembered to test your PORO. From what I see, the argument you pass to the Food class's initialize method is a lot bigger than the example you have on lines 5-10. It may be good to get some fixture data from one food element in the JSON response, and use that here in the test.